src/symbol.ts

```diff
@@ -52,7 +49,20 @@ export class BSONSymbol extends BSONValue {
   }
 
   /** @internal */
-  [Symbol.for('nodejs.util.inspect.custom')](): string {
-    return this.inspect();
+  [Symbol.for('nodejs.util.inspect.custom')](
```

Comment:
Just confirming - were we planning on removing this custom implementation?

Reply:
Yes, I for some reason didn't catch this comment in my first pass, but I'll make the change.